src/js/components/PoliticianListRoot/PoliticianCardForList.jsx

```diff
@@ -1,22 +1,24 @@
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
-import CardForListBody from '../../common/components/CardForListBody';
-import { getTodayAsInteger } from '../../common/utils/dateFormat';
-import { renderLog } from '../../common/utils/logging';
-import CampaignSupporterStore from '../../common/stores/CampaignSupporterStore';
-import CandidateStore from '../../stores/CandidateStore';
-import PoliticianStore from '../../common/stores/PoliticianStore';
-import keepHelpingDestination from '../../common/utils/keepHelpingDestination';
-import { mostLikelyCandidateDictFromList } from '../../utils/candidateFunctions';
+import React, { Component } from "react";
```

Comment:
Hello @StevenSmith-code it looks like your editor/code-clean-up is not using the AirBNB ES Lint rules we have woven into our repository, because we standardize on single quotes. 
I would recommend from the command-line: npm run lint
...to see if there are any errors in the files you are checking in.